README.md

```diff
@@ -72,6 +72,16 @@ Run all tests:
 ./gradlew clean test
 ```
 
+Run tests against local cluster:
+```bash
+./gradlew integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=docker-cluster -Dsecurity=true -Dhttps=true -Duser=admin -Dpassword=admin
```

Comment:
Should we make it a requirement to pass the common-utils version every-time this script is run? or should we set a default value in case it is not passed?

Reply:
@saratvemulapalli Why do we need the common-utils version?  AFAIK the security plugin does not depend on common utils - is there some runtime dependency that is getting pulled in for the tests?

Reply:
@peternied take a look at: https://github.com/opensearch-project/security/pull/1958#discussion_r927031156

Reply:
Following other plugins using the common Config constants, I added this as dependency in build.gradle to be able to use it throughout our plugin. But if it is becoming too complex to use it, I can replace it with the variable names and remove the dependency completely. Suggestions?